source/github-helpers/index.ts

```diff
@@ -47,7 +47,7 @@ export const getOwnerAndRepo = (): {
 };
 
 export function getForkedRepo(): string | undefined {
-	return select<HTMLAnchorElement>('.fork-flag a')?.pathname.slice(1);
+	return select<HTMLAnchorElement>('.fork-flag a')?.pathname.slice(1).split('/', 2).join('/');
```

Comment:
```suggestion
	return select<HTMLAnchorElement>('.fork-flag .text a')?.textContent!.trim();
```

I think this would work here too, no?

Reply:
I guess. Double-check by `blam`ing the line

Reply:
Also you could use the same meta tags I used in the `isForkedRepo` since they're available earlier on the page:

```
<meta name="octolytics-dimension-repository_parent_nwo" content="sindresorhus/refined-github" />
```

```suggestion
	return select<HTMLMetaElement>('[name="octolytics-dimension-repository_parent_nwo"]')?.content;
```

_Originally posted by @fregante in https://github.com/sindresorhus/refined-github/pull/2880_

Reply:
Heh, good job. That's what happens when I review after a merge PR, it gets lost

Reply:
I think this should be put back?

Reply:
I think this should be put back?